### ISSUE TITLE
apa6_word: There are multiple labels on one line

### ISSUE THREAD

**Issue by ericpgreen** [crsh/papaja#170]
I'm having the same issue outputting to word with figure captions as described on [SO](https://stackoverflow.com/questions/47260537/fig-caption-stops-r-markdown-to-docx-when-using-bookdownword-document2). The SO question is about Bookdown, but I think the issue might be the same.

**Comment by ericpgreen** [comment]
Reproduce by creating a new Rmarkdown file from template, changing output to apa6_word, and inserting:

See Figure \ref{myfig}.

````
```{r, fig.cap="My figure//label{myfig}"}
plot(iris)
```
````

**Comment by mariusbarth** [comment]
Hi Eric, using back-slashes instead instead of forward-slashes did the trick on my machine:

    ```{r, fig.cap="My figure\\label{myfig}"}
    plot(iris)
    ```

**Comment by crsh** [comment]
The `bookdown`-way to do this would be to use `\@ref(fig:chunk-name)` in text and

~~~markdown
```{r chunk-name, fig.cap="My figure"}
plot(iris)
```
~~~

As pointed out by Yihui Xie on SO, this may not work if you are using `pandoc` 2.0 unless you install the current development version of `bookdown`.

I will leave this issue open to remind me that I will update the `import` once the new `bookdown` version has been released to CRAN.

**Comment by ericpgreen** [comment]
Sorry, silly error on my part in creating the example. I still see the error after correcting to `My figure\\label{myfig}`, however. The same code used to work on my machine before I upgraded to pandoc 2.0.5 the other day. The output to PDF remains ok, it's just going to word which causes the problem. @mariusbarth are you running pandoc 2?

@crsh is there a way to use `bookdown` and the `papaja` template? This might be a silly question.

**Comment by crsh** [comment]
@ericpgreen `papaja` is built on `bookdown`, so yes, all the usual `bookdown` syntax should work. :wink: But as I pointed out, you may need to install the current development version of `bookdown` to fix incompatibilities with `pandoc` 2.0 (see the SO question you linked to).

**Comment by ericpgreen** [comment]
Got it. That makes more sense now :)

The update fixed the problem. Thanks.

**Comment by crsh** [comment]
It [appears](https://github.com/rstudio/bookdown/releases/tag/v0.6) this issue will be addressed in version 0.6 of `bookdown`.
